src/components/Calculator.js

```diff
@@ -1,111 +1,134 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
 import './UI.css';
+import calculate from './logic/Calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
-    super();
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      // operation: null,
```

Comment:
- [ ] Kindly remove all the commented out lines of code here to make you code cleaner and professional :recycle:

Reply:
Done